script.js

```diff
@@ -8,160 +50,117 @@ function shuffleArray(array) {
         array[j] = temp;
     }
 }
-fetch('./films.json')
-    .then((response) => response.json())
-    .then((json) => {
-        const filmTitles = json.films
-        const films = json.films
-        playButton.addEventListener('click', () => playGame(filmTitles, films))
-    })
 
-let score = 0
-const scoreBoxElem = document.querySelector('.scoreBox')
+const checkAnswer = (button) => {
+    button.addEventListener('click', () => {
+        if (button.textContent === correctAnswer) {
+            button.style.backgroundColor = '#04ac04'
+            score++
+            scoreBoxElem.textContent = score
+
+            if (score % 5 === 0) {
+                confetti({
+                    particleCount: 100,
+                    spread: 70,
+                    origin: {y: .6}
+                })
+            }}
+        else if (button.textContent === wrongAnswer1 || button.textContent === wrongAnswer2) {
+            button.style.backgroundColor = 'red'
+        }
+
+        const interval = setInterval(() => {
+            button.style.backgroundColor = '#efefef'
+            clearInterval(interval)
+            if (films.length >= 3) {
+                playGame(filmTitles, films)
+            } else {
+                endGame()
+            }
+        }, 200)
+        shuffleArray(filmTitles)
+    })
+}
 
 const playGame = (filmTitles, films) => {
```

Comment:
Arguably another poorly-named function - is this playing a game or playing a round of the game?